src/app/zeko/zkapps_rollup.ml

```diff
@@ -750,18 +755,15 @@ module Make (T' : Transaction_snark.S) = struct
       Deferred.t )
     Pickles.Prover.t
 
-  let process_transfer ~is_new ~pointer ~before ~after ~transfer ~vk_hash
-      ~public_key (prover : transfer_prover) =
+  let process_transfer ~is_new ~pointer ~before ~after ~(transfer : TR.t)
+      ~vk_hash ~public_key (prover : transfer_prover) =
     let before = List.map ~f:(value_to_actions TR.typ) before in
     let after = List.map ~f:(value_to_actions TR.typ) after in
     let%bind trans1 =
-      Action_state_extension.prove ~dummy:is_new ~source:pointer before
+      Action_state_extension.prove ~dummy:is_new
+        ~source:Zkapp_account.Actions.empty_state_element before
```

Comment:
pretty sure this is somewhat wrong, somewhat right, but it isn't worth it I think